src/main/java/org/xbill/DNS/Type.java

```diff
@@ -9,181 +9,184 @@
+ */
 public static final int NIMLOC		= 32;
 
-/** Server selection */
+/** {@link SRVRecord Server selection} */
 public static final int SRV		= 33;
 
 /** ATM address */
 public static final int ATMA		= 34;
 
-/** Naming authority pointer */
+/** {@link NAPTRRecord Naming authority pointer} */
 public static final int NAPTR		= 35;
 
-/** Key exchange */
+/** {@link KXRecord Key exchange} */
 public static final int KX		= 36;
 
-/** Certificate */
+/** {@link CERTRecord Certificate} */
 public static final int CERT		= 37;
 
-/** IPv6 address (experimental) */
+/** {@link A6Record IPv6 address (historic)} */
 public static final int A6		= 38;
 
-/** Non-terminal name redirection */
+/** {@link DNAMERecord Non-terminal name redirection} */
 public static final int DNAME		= 39;
 
-/** Options - contains EDNS metadata */
+/** {@link OPTRecord Options - contains EDNS metadata} */
 public static final int OPT		= 41;
 
-/** Address Prefix List */
+/** {@link APLRecord Address Prefix List} */
 public static final int APL		= 42;
 
-/** Delegation Signer */
+/** {@link DSRecord Delegation Signer} */
 public static final int DS		= 43;
 
-/** SSH Key Fingerprint */
+/** {@link SSHFPRecord SSH Key Fingerprint} */
 public static final int SSHFP		= 44;
 
-/** IPSEC key */
+/** {@link IPSECKEYRecord IPSEC key} */
 public static final int IPSECKEY	= 45;
 
-/** Resource Record Signature */
+/** {@link RRSIGRecord Resource Record Signature} */
 public static final int RRSIG		= 46;
 
-/** Next Secure Name */
+/** {@link NSECRecord Next Secure Name} */
 public static final int NSEC		= 47;
 
-/** DNSSEC Key */
+/** {@link DNSKEYRecord DNSSEC Key} */
 public static final int DNSKEY		= 48;
 
-/** Dynamic Host Configuration Protocol (DHCP) ID */
+/** {@link DHCIDRecord Dynamic Host Configuration Protocol (DHCP) ID} */
 public static final int DHCID		= 49;
 
-/** Next SECure, 3rd edition, RFC 5155 */
+/** {@link NSEC3Record Next SECure, 3rd edition} */
 public static final int NSEC3		= 50;
 
-/** Next SECure PARAMeter, RFC 5155 */
+/** {@link NSEC3PARAMRecord Next SECure PARAMeter} */
 public static final int NSEC3PARAM	= 51;
 
-/** Transport Layer Security Authentication, draft-ietf-dane-protocol-23 */
+/** {@link TLSARecord Transport Layer Security Authentication} */
 public static final int TLSA		= 52;
 
-/** S/MIME cert association, draft-ietf-dane-smime */
+/** {@link SMIMEARecord S/MIME cert association} */
 public static final int SMIMEA		= 53;
 
-/** Child Delegation Signer, RFC 8078 */
+/** {@link CDSRecord Child Delegation Signer} */
 public static final int CDS		= 59;
 
-/** Child DNSKEY, RFC 8078 * */
+/** {@link CDNSKEYRecord Child DNSKEY} * */
 public static final int CDNSKEY		= 60;
 
-/** OpenPGP Key, RFC 7929 */
+/** {@link OPENPGPKEYRecord OpenPGP Key} */
 public static final int OPENPGPKEY	= 61;
 
-/** Sender Policy Framework (experimental) */
+/** {@link SPFRecord Sender Policy Framework (experimental)} */
```

Comment:
experimental -> historic (or obsoleted?)
or nothing and let the record javadoc explain

Reply:
+1 to let the record javadoc explain.